src/tone.h

```diff
@@ -71,4 +61,23 @@ class Tone {
 	int gray;
 };
 
+inline Tone blend(const Tone& l, const Tone& r) {
```

Comment:
Could I have this instead either

a) As a static/factory function in the Tone or
b) As a method on the Tone object?

Reply:
well everybody always tells you about encapsulation and namespacing, so a free function looks unfamiliar but when you see this as some kind of operator this argumentation makes sense.

Reply:
The missing encapsulation is that we don't have a namespace around liblcf and Player.